### PR TITLE
Deprecate RSA NetWitness 11.1

### PR DESCRIPTION
**Pull request by ostolero**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://jira-hq.paloaltonetworks.local/browse/CIAC-2061)

## Description
Deprecate RSA NetWitness 11.1 as it is no longer supported.
Also documented the changes between 11.5 and 11.1 in README and remove 11.5 from beta.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/RSANetWitness_v11_1/ReleaseNotes/3_0_0.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+- **RSA NetWitness v11.1 (Deprecated)**
+- Deprecated
```
Comment (ShahafBenYakir) [inline Packs/RSANetWitness_v11_1/ReleaseNotes/3_0_0.md:4]:
```suggestion
- Deprecated. Use RSANetWitnessv115 instead
```

File: Packs/RSANetWitness_v11_1/ReleaseNotes/3_0_0.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+- **RSANetWitnessv115**
+- Removed Beta flag
```
Comment (ShahafBenYakir) [inline Packs/RSANetWitness_v11_1/ReleaseNotes/3_0_0.md:6]:
```suggestion
- Removed integration from Beta.
```

File: Packs/RSANetWitness_v11_1/Integrations/RSANetWitnessv115/RSANetWitnessv115.yml [hunk 3]
```diff
@@ -55,7 +55,7 @@ description: The RSA NetWitness integration provides system log, network, and en
   visibility for real-time collection, detection, and automated response with the
   Cortex XSOAR Enterprise platform. Using full session analysis, customers can extract
   critical data and effectively run security operations automated playbooks.
-display: RSANetWitnessv115 (Beta)
+display: RSANetWitnessv115
```
Comment (bakatzir) [inline Packs/RSANetWitness_v11_1/Integrations/RSANetWitnessv115/RSANetWitnessv115.yml:58]:
```suggestion
display: RSANetWitness v11.5
```

File: Packs/RSANetWitness_v11_1/Integrations/RSANetWitness_v11_1/RSANetWitness_v11_1.yml [hunk 4]
```diff
@@ -49,9 +49,10 @@ configuration:
   name: fetch_limit
   required: false
   type: 0
-description: RSA NetWitness Platform provides systems Logs, Network, and endpoint visibility for real-time collection, detection, and automated response with the Demisto Enterprise platform. Providing full session analysis, customers can extract critical data and effectively operate security operations automated playbook.
-display: RSA NetWitness v11.1
+description:  Deprecated. Use RSANetWitnessv115 instead
```
Comment (bakatzir) [inline Packs/RSANetWitness_v11_1/Integrations/RSANetWitness_v11_1/RSANetWitness_v11_1.yml:52]:
```suggestion
description:  Deprecated. Use RSA NetWitness v11.5 instead
```

**Review by bakatzir — COMMENTED** [review]
@ShirleyDenkberg Can I ask you to breakdown the v11.1 to v11.5 differences?

**Comment by ShirleyDenkberg** [comment]
@bakatzir @ShahafBenYakir I reviewed and edited the  v11.1 to v11.5 differences section as requested.

**Review by ShahafBenYakir — APPROVED** [review]

**Pull request merged**